### PR TITLE
fix: update Session.sendMessage message argument types

### PR DESCRIPTION
**Pull request by eowino**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/cast/docs/reference/chrome/chrome.cast.Session#sendMessage
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by fredrikhr — APPROVED** [review]
LGTM :+1:

File: types/chrome/chrome-cast/index.d.ts [hunk 1]
```diff
@@ -373,7 +373,7 @@ declare namespace chrome.cast {
          */
         sendMessage(
             namespace: string,
-            message: string,
+            message: string | Object,
```
Comment (rbuckton) [inline types/chrome/chrome-cast/index.d.ts]:
```suggestion
            message: string | object,
```
If you use `Object` then pretty much anything but `null` and `undefined` are assignable to `message`.

**Review by rbuckton — CHANGES_REQUESTED** [review]

**Comment by eowino** [comment]
@couven92 Hi, do you mind reviewing the requested changes please?

**Review by fredrikhr — APPROVED** [review]
LGTM 👍

**Comment by eowino** [comment]
@rbuckton Does this need your approval to be moved along?

**Review by matcarlson — APPROVED** [review]

**Comment by matcarlson** [comment]
@rbuckton - It looks like this may be stuck on resolving your change request. Is there something that you see (e.g. submit the change or revert the change as @eowino has made the changes already)

@eowino - Do you see an option to approve the change that @rbuckton requested? If you approve it (even though it's the exact same as your change now), that may allow it to proceed.

**Comment by eowino** [comment]
@matcarlson Unfortunately, I no longer see that option as I approved it on the 11th of June. It seems the only actions available to me are either closing or commenting on this PR.

**Comment by matcarlson** [comment]
If you want to, you can resubmit this PR, and mention me. I'll approve
right away. Not sure how else to get it integrated :\

**Comment by eowino** [comment]
@matcarlson Sure, thanks! Let's try that.

**Comment by eowino** [comment]
New PR as per last comment - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37332

**Pull request closed without merging**